python/lsst/sims/maf/metrics/calibrationMetrics.py

```diff
@@ -161,6 +163,174 @@ def run(self, dataslice, slicePoint=None):
             result = self.badval
         return result
 
+
+class ParallaxCoverageMetric(BaseMetric):
+    """
+    Check how well the parallax factor is distributed. Subtracts the weighted mean position of the
+    parallax offsets, then computes the weighted mean radius of the points.
+    If points are well distributed, the mean radius will be near 1. If phase coverage is bad,
+    radius will be close to zero.
+
+    For points on the Ecliptic, uniform sampling should result in a metric value of ~0.5.
+    At the poles, uniform sampling would result in a metric value of ~1.
```

Comment:
I think I might be using "parallax factor" and "parallax offset" interchangeably. Or maybe the factor is the magnitude and and the offset is the full vector. 

I think it's a feature that uniform time sampling behaves differently as a function of ecliptic latitude. This way, when people look at the parallax uncertainty metric and think, "hey why is the astrometry not as good on the ecliptic", they can look at this map and realize, "oh, the coverage at full parallax factor isn't as large on the ecliptic".